File: Code Comp Practice/LeetCode 30 Days/april/day25.cs
public class Solution
{
    public bool CanJump(int[] nums)
    {
        int furthest = 0;
        for (int i = 0; i < nums.Length; i++)
        {
            if (i > furthest)
            {
                return false;
            }
            if (i + nums[i] > furthest)
            {
                furthest = i + nums[i];
            }
        }
        return true;
    }
}

File: Code Comp Practice/LeetCode 30 Days/april/day25.c
int canJump(int* nums, int numsSize) {
    int furthest = 0;
    for (int i = 0; i < numsSize; i++) {
        if (i > furthest) {
            return 0;
        }
        if (i + nums[i] > furthest) {
            furthest = i + nums[i];
        }
    }
    return 1;
}
